README.md

```diff
@@ -114,7 +117,9 @@ plugins: {
 			// Function called while the user is zooming
 			onZoom: function({chart}) { console.log(`I'm zooming!!!`); },
 			// Function called once zooming is completed
-			onZoomComplete: function({chart}) { console.log(`I was zoomed!!!`); }
+			onZoomComplete: function({chart}) { console.log(`I was zoomed!!!`); },
+			// Function called when wheel input occurs without modifier key
+			onWheelModifierFailed: function({chart, event}) { console.log(`I didn't start zooming!`); }
```

Comment:
Same here, how about `onZoomRejected`? it would be more generic and could be re-used for other use cases.